docs/topics/testing/overview.txt

```diff
@@ -46,6 +46,17 @@ automatically build a test suite out of those test cases, and run that suite.
 
 For more details about :mod:`unittest`, see the Python documentation.
 
+.. admonition:: Where should the tests live?
+
+    The default :djadmin:`startapp` template will create a ``tests.py`` file
+    in the new application. This might be fine if you only have a few tests,
+    but as your test suite grows you'll likely want to restructure it into a
+    tests module so you can split your tests into different files such as
```

Comment:
... into a tests *package* so you can split your tests into different *submodules* ...?